setup.py

```diff
@@ -124,7 +124,9 @@ def get_version():
 	packages=find_packages(exclude=('tests', 'tests.*')),
 	include_package_data=True,
 	zip_safe=False,
-	install_requires=[],
+	install_requires=[
+		'pytz'
+	],
```

Comment:
Are you saying making the feature altogether optional if they have pytz installed? That makes sense.